### PR TITLE
 Contribute Best Practices for Spreadsheet Data blog post -- Naailah Trejo

### PR DESCRIPTION
**Pull request by FNTrejo8**
A blog post describing the best practices for organizing spreadsheet data, in order for it to be analyzed by R. 
Please take a look, I'd welcome any suggestions. Thanks! @naupaka @melt6457 @mhaliz

### CODE REVIEW

**Comment by naupaka** [comment]
Thanks for the PR! We'll get a formal review going in a bit, but a few quick fixes:
* Please remove the DS_Store file from the PR
* Add your name somewhere in the post
* The alt-text for the images should say what the images are of (e.g. [see here](https://en.wikipedia.org/wiki/Alt_attribute))

**Comment by FNTrejo8** [comment]
Remove DS_Store file: Fixed by b0a4dcb2e98c9c6073d3005a40cf8c957f84440e
Add name and alt-text: Fixed by 852d040bb356ba70ac25976c2301337961f55cd5

**Review by emhumphrey — APPROVED** [review]
Hey Naailah,
This looks great! I found it to be very thorough, concise, and informative! I read through this a couple of times seeing if there was anything I could think of that you could add to improve it and I really think you covered it all. I also really liked that you had so many visual images as examples to represent what you were talking about as well.

**Review by naupaka — CHANGES_REQUESTED** [review]
The post looks great! Please remove the .DS_Store file in the assets directory so that we avoid any merge conflicts. Then this should be ready to merge!

**Comment by FNTrejo8** [comment]
Removed .DS_Store from assets directory, fixed by a1222a8cdb939c368aa9f67fcd85cf7ed0708c58

**Comment by naupaka** [comment]
@FNTrejo8 please fix the merge conflict with the .gitignore file -- you can do it in the web interface or on your local machine

**Comment by FNTrejo8** [comment]
@naupaka Fixed the merge conflicts with the gitignore file by c754af3b71fd8321e64bfe1c2a2393a90ff4fffb and 5975de2cbf3e4635c3297737eda41431c6d08e5d

**Review by naupaka — APPROVED** [review]

**Comment by naupaka** [comment]
👍

**Comment by naupaka** [comment]
Looks great! https://2020f-usfca-btec-640.github.io/spreadsheet,/notes,/column,/dates,/csv/2020/11/02/best-practices-for-spreadsheet-data.html

**Comment by naupaka** [comment]
@FNTrejo8 actually looks like there's an issue -- you shouldn't use commas in your tag list, just spaces. See how it messes up the URL? If you submit a quick PR with that fix and @ me, I'll merge it right away

File: _posts/2020-11-02-best-practices-for-spreadsheet-data.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "Best Practices for Spreadsheet Data!"
+date:   2020-11-02 12:54:04 -0700
+categories: spreadsheet, notes, column, dates, CSV
```
Comment (naupaka) [inline _posts/2020-11-02-best-practices-for-spreadsheet-data.md:5]: Should just use spaces to separate -- otherwise it messes up the tag list

**Pull request merged**